extensions/kafka-client/runtime/pom.xml

```diff
@@ -37,6 +37,11 @@
             <artifactId>quarkus-kubernetes-service-binding</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
```

Comment:
Before merging, we need to think about this.

I wouldn't say I like adding a dependency like this, which would be pulled in any application using the kafka-client, even if strimzi oauth is not used. 
So, we should externalize this as a new extension, potentially on the quarkiverse.

Note that it will also be a problem for productization.

Reply:
That's why this dependency has `provided` scope. Neither itself nor its transitive dependencies will be pulled into projects using this extension.

As discussed, I agree with the productization issues that it can cause.

Reply:
Just checked, you are right! I should use that trick more often.